tremor-cli/src/logging.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020-2021, The Tremor Team
```

Comment:
I think this file can go, we no longer need it :)

Reply:
This file is used in the main.rs